src/engine/ruleset/decoders/aws/s3/aws-s3.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: decoder/aws-s3/0
+
+metadata:
+  title: AWS S3 Decoder
+  description: Decoder for Amazon S3 server access logs
```

Comment:
version?I think it is necessary to emphasize in the name and description that it is an integration and not the original AWS logs.

Reply:
done